src/main/groovy/com/github/sherter/googlejavaformatgradleplugin/GoogleJavaFormatExtension.groovy

```diff
@@ -34,6 +35,13 @@ class GoogleJavaFormatExtension {
         return javaFiles
     }
 
+    void setSortImports(boolean sortImports) {
+        this.sortImports = sortImports
+    }
+
+    boolean getSortImports() {
+      return sortImports
+    }
```

Comment:
The "option stuff" was meant to add some additional checks to prevent, for example, setting the same option twice. I think it provides little value and unnecessarily complicates the implementation, so going with a simple field is fine. You can let Groovy generate the getter and setter by removing the private modifier from the field.